Sources/Build/misc.swift

```diff
@@ -67,7 +68,7 @@ extension ClangModule {
         
         if dirs.isEmpty {
```

Comment:
We should get some comments here explaining the intent of this code w.r.t. what convention it is trying to support / enforce.